### ISSUE TITLE
BUG - Segmentation fault on EndeavourOS (Arch based)

### ISSUE THREAD

**Issue by luciorgomes** [wxGlade/wxGlade#547]
wxGlade don't start on EndeavourOS (Arch based). It's on Python 3.12. 

The execution ends with a segmentation fault:

[luciorg@lucio-dellg155530 bin]$ '/run/media/luciorg/HD Externo/home/bin/wxGlade/wxglade' 
INFO    : Starting wxGlade version "1.1.0a2" on Python 3.12.3
INFO    : Using wxPython 4.2.1
INFO    : Loading "wconfig" modules from /run/media/luciorg/HD Externo/home/bin/wxGlade/widgets/widgets.txt:
INFO    : Load code generators:
INFO    :   python generator loaded
INFO    :   XRC generator loaded
INFO    :   perl generator loaded
INFO    :   lisp generator loaded
INFO    :   C++ generator loaded
INFO    : Loading widgets from /run/media/luciorg/HD Externo/home/bin/wxGlade/widgets/widgets.txt:
INFO    :       frame
INFO    :       dialog
INFO    :       panel
INFO    :       notebook
INFO    :       splitter_window
INFO    :       button
INFO    :       toggle_button
INFO    :       bitmap_button
INFO    :       spin_button
INFO    :       text_ctrl
INFO    :       choice
INFO    :       combo_box
INFO    :       list_box
INFO    :       check_list_box
INFO    :       checkbox
INFO    :       radio_button
INFO    :       radio_box
INFO    :       spin_ctrl
INFO    :       spin_ctrl_double
INFO    :       slider
INFO    :       gauge
20:55:48: Debug: Adding duplicate image handler for 'Windows bitmap file'
20:55:48: Debug: Adding duplicate animation handler for '1' type
20:55:48: Debug: Adding duplicate animation handler for '2' type
INFO    :       calendar_ctrl
INFO    :       generic_calendar_ctrl
INFO    :       datepicker_ctrl
INFO    :       timepicker_ctrl
INFO    :       list_ctrl
INFO    :       tree_ctrl
INFO    :       grid
INFO    :       static_text
INFO    :       hyperlink_ctrl
INFO    :       static_line
INFO    :       static_bitmap
INFO    :       spacer
INFO    :       property_grid_manager
INFO    :       search_ctrl
INFO    :       custom_widget
INFO    :       menubar
INFO    :       toolbar
INFO    :       statusbar
INFO    : Load sizer generators:
INFO    :   for python
INFO    :   for XRC
INFO    :   for perl
INFO    :   for lisp
INFO    :   for C++
Falha de segmentação (imagem do núcleo gravada)

**Comment by wahnfred** [comment]
Hello,

the same happens on openSUSE Tumbleweed

Git revision:  `v1.1.0a2-13-g6372ced0`

```
INFO    : Starting wxGlade version "1.1.0a2" on Python 3.11.9
INFO    : Using wxPython 4.2.1

....

INFO    : Load sizer generators:
INFO    :   for lisp
INFO    :   for perl
INFO    :   for XRC
INFO    :   for python
INFO    :   for C++
Segmentation fault (core dumped)

```

```
Thread 1 "python3" received signal SIGSEGV, Segmentation fault.
0x00007ffff421da19 in ?? () from /lib64/libgtk-3.so.0
```

GTK3 version:  `3.24.41`

Best regards

**Comment by reticulatus** [comment]
See: https://github.com/wxGlade/wxGlade/issues/542

**Comment by wahnfred** [comment]
Hello Richard,

Thank you very much for this information,

> See: #542

[DietmarSchwertberger](https://github.com/DietmarSchwertberger) commented [on Mar 24](https://github.com/wxGlade/wxGlade/issues/542#issuecomment-2016793443)
>> Sorry, but I can't debug a distribution that is in such a bad shape.

Sorry but this much ignorance is hilarious.
Tumbleweed, as the name says, is a rolling-release - so no

`zypper up` 

but

`zypper dup`

I'm switching to QT now.

Best regards.
Glen

**Comment by DietmarSchwertberger** [comment]
A rolling release that does not start to roll hardly deserves its name.

This looks like a bug in the specific gtk version.
Could you please try the latest repository version of wxGlade? The dummy panel for size calculation is now deleted later.

**Comment by wahnfred** [comment]
Hello,

doesn't crash anymore.

Best regards.
Glen

**Comment by DietmarSchwertberger** [comment]
version 1.1.0b1 is online

**Comment by ardovm** [comment]
This fixes issue #550 as well.
